english/marshmallow.md

```diff
@@ -612,4 +612,6 @@ You escape into a [recording studio](more_cowbell/more_cowbell.md)
 
 Now for some [music](https://www.youtube.com/watch?v=Ua2HPQUKJao)
 
-And then you stumble upon something [completely different](http://www.atlasobscura.com/)
\ No newline at end of file
+And then you stumble upon something [completely different](http://www.atlasobscura.com/)
+
+You decide to [take a shortcut](new_movie/new_movie.md) on the way to the movie...
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.